src/PKSim.Infrastructure/Services/PKSimLoggerCreator.cs

```diff
@@ -0,0 +1,44 @@
+﻿using PKSim.Infrastructure.Services;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Console;
+using NHibernate.Mapping;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using PKSim.Infrastructure.Extensions;
+using System.Collections.Concurrent;
+
+public class PKSimLoggerCreator : ILoggerCreator
```

Comment:
This can be in Core

Reply:
@msevestre @georgeDaskalakis I agree. We should move it when we are able to create new versions of our nugets so we could use them in MoBi as well.